src/commons/jira/client.py

```
@@ -0,0 +1,424 @@
+            status = getattr(issue.fields, "status", None)
+            return status.name if status else None
+        if field_name_lower == "assignee":
+            assignee = getattr(issue.fields, "assignee", None)
+            return assignee.displayName if assignee else None
+
+        # Handle custom fields - search by name
+        try:
+            for field in self.jira.fields():
+                if field["name"].lower() == field_name_lower:
+                    return getattr(issue.fields, field["id"], None)
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)